lib/vamble-core/src/vamble_core/event_manager.py

```diff
@@ -27,6 +27,25 @@ def get_event(self, event_id: int):
         if event:
             return self.get_event_model(event)
 
+    def create_event(self, name, description, esport_name, parent_event_name, team_names, ranks):
```

Comment:
Can we change from names -> ids

i.e. esport_name -> esport_id, parent_event_name -> parent_event_id, team_names -> team_ids